developer-guides/rest-api/groups/listAll/README.md

```diff
@@ -0,0 +1,55 @@
+# Group List All
```

Comment:
Please add this file to the [Table of Contents](https://github.com/RocketChat/docs/blob/master/_data/toc.yml)

Reply:
Added.